docs/_documentations/workingwithtemplates.md

```diff
@@ -91,6 +91,8 @@ Add your template sources to Codewind with the **Template Source Manager**.
 1. Add the template.
    - **VS Code:** In the **Template Source Manager**, click **Add New**.
    - **Eclipse:** Right-click the connection in the Codewind Explorer view and select **Manage Template Sources...**. After the **Manage Template Sources** wizard appears, click **Add...**.
+   - **IntelliJ:** Right-click the connection in the Codewind Explorer view and select **Manage Template Sources...**. After the **Manage Template Sources** wizard appears, select **My Private Template Repository** and click **OK**.
 2. Enter the URL and any other information.
    - **VS Code:** Enter the URL to your template source `index` file and click **Enter** to confirm.
-   - **Eclipse:** Fill in the fields for **URL**, **Name**, and **Description**. Click **OK** when you're done.
+   - **Eclipse:** Complete the fields for **URL**, **Name**, and **Description**. Click **OK** when you're done.
+   - **IntelliJ:** Complete the field for **URL**, select an authentication method, and enter the required authentication information. Click **Next** when you're done. An **Authentication Tests** message appears to inform you of a successful test. Click **OK**.
```

Comment:
Edit phrase to something like this:  .....Click Next when you're done to reach the Details page.  Complete the required fields for Name and Description.

Reply:
The Eclipse description is out-of-date.  The wizard was also revamped so that the Details page is last.  I think you can make Eclipse and IntelliJ have the same description.